styles/typography/typography-preset-7.json

```diff
@@ -112,16 +99,23 @@
 			"button": {
 				"typography": {
 					"fontFamily": "var:preset|font-family|literata",
-					"fontWeight": "600",
-					"letterSpacing": "-0.96px",
+					"fontSize": "var:preset|font-size|medium",
```

Comment:
The font size is already set to medium in theme.json > elements > button, and it is the same as on the body, so this can be removed.